cmd/nfd-master/main.go

```diff
@@ -125,6 +125,9 @@ func initFlags(flagset *flag.FlagSet) (*master.Args, *master.ConfigOverrideArgs)
 	flagset.StringVar(&args.Options, "options", "",
 		"Specify config options from command line. Config options are specified "+
 			"in the same format as in the config file (i.e. json or yaml). These options")
+	flagset.DurationVar(&args.ResyncPeriod, "resync-period", 60,
```

Comment:
the default value should be changed to `time.Duration(1)*time.Hour` and description should be changed that it accepts duration string

Reply:
Yeah I know, I actually questioned that in the issue section, but since I didn't get any response, I used `Moment` as the resync period.